tests/instruments/jobinyvon/test_jobinyvon270m_with_device.py

```diff
@@ -0,0 +1,131 @@
+import time
+from pymeasure.instruments.jobinyvon.jobinyvon270m import JY270M
+from pyvisa.constants import ControlFlow, Parity, StopBits
+import pytest
+
+
+@pytest.fixture(scope="module")
+def init_communication():
+    instr = JY270M('COM1',
+                   baud_rate=9600,
+                   timeout=300,
+                   parity=Parity.none,
+                   data_bits=8,
+                   stop_bits=StopBits.one,
+                   flow_control=ControlFlow.dtr_dsr,
+                   write_termination='',
+                   read_termination='',
+                   includeSCPI=False)
+    return instr
```

Comment:
See our documentation on how to write [device tests](https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments/tests.html#device-tests).
For we have a fixture, which allows to give the resource name via command line parameter as not everyone has their instrument at COM1.

Reply:
We changed 'COM1' for 'connected_device_address', and inside the input of the function: 
def init_communication(connected_device_address):

We don't know if this is the correct way to do it but it is what we found in the documentation.